bluetooth/src/commonMain/kotlin/scanner/Scanner.kt

```diff
@@ -97,8 +98,8 @@ abstract class BaseScanner constructor(
     protected val autoRequestPermission: Boolean = settings.autoRequestPermission
     internal val autoEnableSensors: Boolean = settings.autoEnableSensors
 
-    protected val sharedEvents = SequentialMutableSharedFlow<Scanner.Event>(0, settings.eventBufferSize, coroutineScope)
-    override val events: Flow<Scanner.Event> = sharedEvents.asSharedFlow()
+    protected val sharedEvents = Channel<Scanner.Event>(UNLIMITED)
```

Comment:
Same here `eventBufferSize` can be removed from the `Settings`?

Reply:
Yes, good point, will do that in a downstream PR.